r4/fhir_context.bal

```diff
@@ -24,6 +24,11 @@ public isolated class FHIRContext {
     private boolean inErrorState = false;
     private int errorCode = 500;
     private string rawPath = "";
+    private PaginationContext paginationContext = {
+        paginationEnabled: true,
+        pageSize: 10,
```

Comment:
we are hard coding pageSize 10 in two places?

Reply:
fixed